code/modules/organs/organ.dm

```diff
@@ -127,6 +127,22 @@
 	blood_DNA[dna.unique_enzymes] = dna.b_type
 	set_species(dna.species)
 
+/obj/item/organ/proc/set_bodytype(decl/bodytype/new_bodytype)
+	if(isnull(new_bodytype))
+		CRASH("Null bodytype passed to set_bodytype!")
+	if(ispath(new_bodytype, /decl/bodytype))
+		new_bodytype = GET_DECL(new_bodytype)
+	if(!istype(new_bodytype))
+		CRASH("Invalid bodytype [new_bodytype]")
+	vital_to_owner = null
```

Comment:
Why this?

Reply:
GET_DECL can return null for abstract bodytypes that still pass the ispath check above.